docs/DevGuide/Variables.md

```diff
@@ -0,0 +1,75 @@
+\cond NEVER
+Distributed under the MIT License.
+See LICENSE.txt for details.
+\endcond
+# Using Variables in SpECTRE
+
+# What is a Variables and Why Use Them?
+Variables are a data structure that allow you to make most if not all memory
+allocations needed at one time. Allocating memory can be expensive as it holds
+up any computations while the memory is being allocated. Using a Variables to
+allocate memory before it's needed allows the computation to operate smoothly
+uninteruppted.
+
+# Defining a Variables
+To define a Variables, you'll need to
+`#include "DataStructures/Tags/TempTensor.hpp"` this will give you access to
+temporary Scalars and Tensors we'll need to allocate. Once you have that,
+you can define a Variables that allocates one Scalar like this
+```
```

Comment:
I _think_ if you put ` cpp` after the triple tick you get C++ syntax highlighting. same comment below

Reply:
did you try this?

Reply:
oh whoops, I missed this comment, it does work!